.github/pull_request_template.md

```diff
@@ -1,9 +1,9 @@
-
```

Comment:
@JosephTLyons Can we leave these blank lines up top?

They're there to leave room for the description above the release notes without having to press <kbd>Enter</kbd> twice.

Reply:
Gotcha, yeah I'll add them back.